src/Symfony/Component/HttpFoundation/InputBag.php

```diff
@@ -72,12 +72,12 @@ public function add(array $inputs = [])
     /**
      * Sets an input by name.
      *
-     * @param string|array $value
+     * @param string|array|null $value
      */
     public function set(string $key, $value)
     {
-        if (!is_scalar($value) && !\is_array($value) && !method_exists($value, '__toString')) {
-            trigger_deprecation('symfony/http-foundation', '5.1', 'Passing "%s" as a 2nd Argument to "%s()" is deprecated, pass a string or an array instead.', get_debug_type($value), __METHOD__);
+        if (null !== $value && !is_scalar($value) && !\is_array($value) && !method_exists($value, '__toString')) {
```

Comment:
@taylorotwell can I suggest `!is_null($value)`

Reply:
I'm just following the convention elsewhere in this file.